src/Compilers/VisualBasic/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.vb

```diff
@@ -556,6 +556,163 @@ True
 ]]>
 
             Dim verifier As CompilationVerifier = CompileAndVerify(source, expectedOutput)
```

Comment:
The description of the change says an exception is thrown by the instrumented code. Why wasn't an exception thrown in the call to `CompileAndVerify()` in this test before your change? Is there a test that would throw?